dbt_project/air_dbt/README.md

```diff
@@ -0,0 +1,15 @@
+Welcome to your new dbt project!
```

Comment:
The dbt project should be living in `dbt_project`, not `dbt_project/air_dbt`.

Reply:
Ok.